pkg/catalog/search.go

```diff
@@ -9,11 +9,21 @@ import (
 )
 
 // Search queries the catalog at catalogURL for profiles matching the provided searchName.
+// If no searchname is provided it returns all profiles
 func Search(catalogClient CatalogClient, searchName string) ([]profilesv1.ProfileCatalogEntry, error) {
-	q := map[string]string{
-		"name": searchName,
+	var data []byte
+	var statusCode int
+	var err error
+
+	if searchName == "all" {
+		data, statusCode, err = catalogClient.DoRequest("/profiles", nil)
+	} else {
```

Comment:
does this mean `pctl search all` is equivalent to `pctl search --all`? Also it blocks users from searching for the string `all`, which might be fine if its intentional :smile:

Reply:
i think this was not intentional, i'm gonna remove this. nice catch 👌🏻